cypress/e2e/cloud/about.test.ts

```diff
@@ -46,7 +46,13 @@ describe.skip('About Page for free users with only 1 user', () => {
 })
 
 describe('About Page for free users with multiple users', () => {
-  beforeEach(() =>
+  beforeEach(() => {
+    cy.intercept('GET', 'api/v2/quartz/identity', req => {
+      req.continue(res => {
+        res.body.user.orgCount = 1
+        res.send(res.body)
```

Comment:
The default state in quartz mock is more than one org, so we have to modify it here to ensure the test for a 'default' free account (which has only one org) works correctly.